### PR TITLE
Feature/palindrome

### PR DESCRIPTION
**Pull request by EwiJosepha**

### CODE REVIEW

File: assets/scripts/script.js [hunk 1]
```diff
@@ -1,27 +1,32 @@
...
+  let result = ''
+  if (enterr === '') {
+    displayy.innerHTML = ' type a text or number'
+  } else {
+    const restrictSymbols = enterr.replace(/[^a-z0-9]/g, '').toLowerCase()
+    result = restrictSymbols.split('').reverse().join('')
 
-//   let result = "";
-//   result = enterr.split("").reverse().join("");
+    // console.log('this is result', result)
```
Comment (DanCarl857) [inline assets/scripts/script.js]: remove dead code in your final submission

File: assets/scripts/script.js [hunk 2]
```diff
@@ -1,27 +1,32 @@
...
+      displayy.innerHTML = 'it is a palindrome'
+      console.log('it is a palindrome')
 
-//   if (enterr === result) {
-//     displayy.innerHTML = "it is a palindrome";
-//     console.log("it is a palindrome");
+      // return "it is a palindrome"
+    } else {
+      displayy.innerHTML = 'it is not a palindrome'
+      console.log('it is not a palindrome')
```
Comment (DanCarl857) [inline assets/scripts/script.js:23]: I'm guessing your consoles are for debugging purposes? remove them in the final submission

**Review by DanCarl857 — CHANGES_REQUESTED** [review]
This is a really great attempt.
But could you try `Do geese see God`

Great stuff. Your UI is superb

File: assets/scripts/script.js (hunk 1 above)
Comment (EwiJosepha) [reply assets/scripts/script.js]: okay sir, thank you

**Comment by gastonche** [comment]
Hi @EwiJosepha — review verdict: **Rejected** (grade 0/5) — and I want to be clear up front that this is a hard rule call, not a knock on your effort. The UI here is genuinely lovely.

Two decisive issues:

1) **Regex isn't allowed for this exercise, and using one voids the submission.** The brief: "Do not use regular expressions... Any use of regular expressions will render your submission void." In `assets/scripts/script.js`:
```js
const restrictSymbols = enterr.replace(/[\s\w\d_]/gi, '').toLowerCase()
```
`/[\s\w\d_]/gi` is a regular expression, so per the project's own rule this is void. Clean the string without regex — loop the characters and keep only letters, e.g. keep `ch` where `ch.toLowerCase() >= 'a' && ch.toLowerCase() <= 'z'`.

2) **The palindrome logic is inverted**, so it reports almost everything as a palindrome. `[\s\w\d_]` matches letters/digits/underscore/whitespace, so `.replace(...,'')` deletes all the letters and keeps only symbols. For ordinary text the cleaned string is empty, and `'' === ''` reversed is always true. I tested the deployed app: **"hello" → "it is a palindrome"** and **"banana" → "it is a palindrome"** (both wrong). Fix: KEEP the letters (lowercased), then compare that string to its reverse.

What's good: the UI/UX is great (ticked), the empty-input guard is a nice touch, and the PR (feature/palindrome → main) and folder structure are clean.

Next steps: drop the regex, build the cleaned string by keeping letters via a loop/String methods, compare to its reverse. With the UI you already have, that's an easy 4–5 on resubmit. — Gaston

**Pull request merged**